### ISSUE TITLE
TM hangs with missing profile parameter

### ISSUE THREAD

**Issue by jhg03a** [apache/trafficcontrol#2876]
if you build a server profile that doesn't contain a parameter like below and assign it to a cache, TM will serve port 80, but will loop forever trying to poll that cache and not come all the way up.
```
Name: health.polling.url
Config File: rascal.properties
Value: http://${hostname}/_astats?application=&inf.name=${interface_name}
```

**Comment by mitchell852** [comment]
@jrushford is on it

**Comment by jrushford** [comment]
Resolved with #2921
